### DISCUSSION TITLE
Godot guix

### DISCUSSION THREAD

**Discussion by DeanLemans** [godotengine/godot-proposals#11050] [Platforms]
A Godot guix package.
Could (also) be for beta and rc.

Just putting this here to jumpstart a discusion.

**Comment by Calinou** [comment]
This can be done by Guix packagers themselves. You don't need to ask upstream developers about it :slightly_smiling_face: 

As far as I know, none of the core Godot developers use Guix, so we wouldn't be able to provide extra guidance anyway.

**Comment by ArchercatNEO** [comment]
Guix already has a (third-party)[https://packages.guix.gnu.org/packages/godot/4.3/] godot package.

Godot tries very very hard to be a widely compatible binary you can just download and it works (as a nix user I've tried very hard to *un*vendor a lot of dependencies, it's a lot harder than you'd think) and unless the linux ecosystem consolidates on one package format (highly unlikely anytime soon) a binary with bundled vendored libs is really the only simple way for godot to be distributed. If the godot maintainers were to create a first party package it would almost certainly be a flatpak or an appimage. In any case godot doesn't really need to be packaged on most systems as it depends on libs you almost certainly have installed. NixOS (and I assume guix uses the store?) doesn't just work because of the nix store but it's been packaged already. If on guix you can't just download godot and it runs (which is valid for a linux distro) then maintenance of godot is something to bring up with the package maintainers for guix as that system will have some requirements that godot doesn't support despite it's best efforts. 

If you want beta or rc versions of godot or the package is broken then that is something to bring up with the package maintainers of guix not with godot.
